### PR TITLE
Product update on 07.04.2023

### PR DESCRIPTION
**Pull request by MartaD**

### CODE REVIEW

File: docs/change-log.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update April 7th, 2023
+
+- **Improved:** [Driver inputs](https://docs.humanitec.com/reference/concepts/resources/definitions#driver-inputs) are now validated against Driver Definition Schemas to catch misconfiguration errors in [resource definitions](https://docs.humanitec.com/reference/concepts/resources/definitions) sooner.
```
Comment (MartaD) [inline docs/change-log.md:7]:
```suggestion
- **Improved:** [Driver inputs](https://docs.humanitec.com/reference/concepts/resources/definitions#driver-inputs) are now validated against Driver Definition Schemas to catch misconfiguration errors in [resource definitions](https://docs.humanitec.com/reference/concepts/resources/definitions) earlier.
```

File: docs/change-log.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update April 7th, 2023
+
+- **Improved:** [Driver inputs](https://docs.humanitec.com/reference/concepts/resources/definitions#driver-inputs) are now validated against Driver Definition Schemas to catch misconfiguration errors in [resource definitions](https://docs.humanitec.com/reference/concepts/resources/definitions) earlier.
+- **Fixed:** Special character "*" in [matching criteria](https://docs.humanitec.com/reference/concepts/resources/definitions#matching-criteria) added via API is now correctly interpreted as "all values".
+- **Fixed:** [Deployment diff](https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments) is now rendered correctly for past deployments with null properties.
+- **Fixed:** Setting Workload Variable's value to an empty string no longer renders as a deleted key in the [deployment diff](https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments) view.
```
Comment (MartaD) [inline docs/change-log.md:10]:
```suggestion
- **Fixed:** Setting [Workload Variable's](https://docs.humanitec.com/using-humanitec/work-with-workloads/work-with-environment-variables) value to an empty string no longer renders as a deleted key in the [deployment diff](https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments) view.
```

File: docs/change-log.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update April 7th, 2023
+
+- **Improved:** [Driver inputs](https://docs.humanitec.com/reference/concepts/resources/definitions#driver-inputs) are now validated against Driver Definition Schemas to catch misconfiguration errors in [resource definitions](https://docs.humanitec.com/reference/concepts/resources/definitions) earlier.
+- **Fixed:** Special character "*" in [matching criteria](https://docs.humanitec.com/reference/concepts/resources/definitions#matching-criteria) added via API is now correctly interpreted as "all values".
```
Comment (sujaya-sys) [inline docs/change-log.md:8]: we could link to the API docs here instead https://api-docs.humanitec.com/#tag/MatchingCriteria

Comment (sujaya-sys) [reply docs/change-log.md:8]: The wording could be a bit more specific I think, for example

"When selecting an asterisk `*` as a matching criterium via the API, it is now correctly interpreted as..."

File: docs/change-log.md [hunk 4]
```diff
@@ -2,6 +2,13 @@
...
+- **Fixed:** [Deployment diff](https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments) is now rendered correctly for past deployments with null properties.
+- **Fixed:** Setting [Workload Variable's](https://docs.humanitec.com/using-humanitec/work-with-workloads/work-with-environment-variables) value to an empty string no longer renders as a deleted key in the [deployment diff](https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments) view.
```
Comment (sujaya-sys) [inline docs/change-log.md:10]: * Setting "a" Workload...

File: docs/change-log.md [hunk 5]
```diff
@@ -2,6 +2,13 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update April 7th, 2023
+
+- **Improved:** [Driver inputs](https://docs.humanitec.com/reference/concepts/resources/definitions#driver-inputs) are now validated against Driver Definition Schemas to catch misconfiguration errors in [resource definitions](https://docs.humanitec.com/reference/concepts/resources/definitions) earlier.
```
Comment (sujaya-sys) [inline docs/change-log.md:7]: would scratch the "earlier"

Comment (MartaD) [inline docs/change-log.md:7]:
```suggestion
- **Improved:** [Driver inputs](https://docs.humanitec.com/reference/concepts/resources/definitions#driver-inputs) are now validated against Driver Definition Schemas to catch misconfiguration errors in [resource definitions](https://docs.humanitec.com/reference/concepts/resources/definitions.
```

File: docs/change-log.md (hunk 3 above)
Comment (MartaD) [inline docs/change-log.md:8]:
```suggestion
- **Fixed:** Selecting an asterisk `*` as a [matching criterium](https://api-docs.humanitec.com/#tag/MatchingCriteria) via the API  is now correctly interpreted as "all values".
```

I'm not sure the "when" works well with "it is interpreted" structure. How about this?

File: docs/change-log.md (hunk 4 above)
Comment (MartaD) [inline docs/change-log.md:10]:
```suggestion
- **Fixed:** Setting s [Workload Variable's](https://docs.humanitec.com/using-humanitec/work-with-workloads/work-with-environment-variables) value to an empty string no longer renders as a deleted key in the [deployment diff](https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments) view.
```

File: docs/change-log.md (hunk 3 above)
Comment (sujaya-sys) [reply docs/change-log.md:8]: Looks good :)

File: docs/change-log.md [hunk 6]
```diff
@@ -2,6 +2,13 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update April 7th, 2023
+
+- **Improved:** [Driver inputs](https://docs.humanitec.com/reference/concepts/resources/definitions#driver-inputs) are now validated against Driver Definition Schemas to catch misconfiguration errors in [resource definitions](https://docs.humanitec.com/reference/concepts/resources/definitions).
+- **Fixed:** Selecting an asterisk `*` as a [matching criterium](https://api-docs.humanitec.com/#tag/MatchingCriteria) via the API  is now correctly interpreted as "all values".
+- **Fixed:** [Deployment diff](https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments) is now rendered correctly for past deployments with null properties.
```
Comment (sujaya-sys) [inline docs/change-log.md:9]: I'm not sure if I asked this before (can't find my comment): by "null" properties you mean empty values?

**Review by sujaya-sys — APPROVED** [review]
Added one more comment. Looks good to me otherwise 👍

**Comment by sujaya-sys** [comment]
@rachfop we thought it might be helpful for you to have a look at our product updates going forward. Just in case you were wondering why Marta added you as a reviewer :)

File: docs/change-log.md (hunk 6 above)
Comment (MartaD) [reply docs/change-log.md:9]: I meant properties with the value set to "null", but it probably isn't that important here, I'll change it to empty

Comment (MartaD) [inline docs/change-log.md:9]:
```suggestion
- **Fixed:** [Deployment diff](https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments) is now rendered correctly for past deployments with empty properties.
```

**Review by chrishumanitec — APPROVED** [review]
LGTM

**Pull request merged**